boards.txt

```diff
@@ -48860,8 +48860,8 @@ huidu_hd_wf2.build.partitions=default
 huidu_hd_wf2.build.defines=
 huidu_hd_wf2.build.loop_core=
 huidu_hd_wf2.build.event_core=
-huidu_hd_wf2.build.flash_type=qio
-huidu_hd_wf2.build.psram_type=qspi
+huidu_hd_wf2.build.flash_type=dio
+huidu_hd_wf2.build.psram_type=
```

Comment:
Please do not edit those lines or board will never compile. flash and psram types are required and not meaning that there is psram or that the flash mode could not be changed to DIO. Same for the other board

Reply:
oh, that is unexpected... then I assume that I could just leave it as-as and it would be fine 
tnx!

Reply:
those two lines control which libs to be used and were added to accommodate OPI flash and psram on S3